plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/font-pairing-variations/font-families-loader.tsx

```diff
@@ -38,12 +38,6 @@ export const FontFamiliesLoader = ( {
 					`${ fontFamily }:${ FONT_AXIS }`,
 				] ),
 				[ 'display', 'swap' ],
-				[
```

Comment:
This fixes the characters not loaded in preview.

cc @chihsuan since you've implemented the initial optimization. ChatGPT recommended using dynamic unique character detection based on content, but I think that's non-trivial. I'm not sure if we can have any variation of this optimization.

Reply:
It slipped my mind when implementing the optimisation that these fonts are also used in the editor panel, thanks for fixing this!